metadata.rb

```diff
@@ -20,5 +20,4 @@
 depends          'osl-nginx'
 depends          'osl-nodejs'
 depends          'osl-repos'
-depends          'systemd', '~> 3.2.4'
-depends          'user'
+depends          'users', '~> 7.0'
```

Comment:
Can we please keep this PR only focused on the systemd feature and leave out the user -> users? I'd like to deal with that in a separate PR please.